react-d3-chart/src/components/LineChart/LineChart.stories.tsx

```diff
@@ -18,7 +18,7 @@ ranges.split(',').forEach((range) => {
     }));
     data.push({
       coordinates: [...temp],
-      colour:
+      color:
         '#' + (0x1000000 + Math.random() * 0xffffff).toString(16).substr(1, 6),
```

Comment:
I recommend using a template string for this.

```
`#${(0x1000000 + Math.random() * 0xffffff).toString(16).substr(1, 6)}}`
```

Reply:
I've swapped the color to use d3 color schemes in the victory graphs. The regular d3 graphs will get deleted after I've implemented all the required features in d3.